apps/server/src/modules/board-context/board-context-api-helper.service.ts

```diff
@@ -35,4 +52,47 @@ export class BoardContextApiHelperService {
 		/* istanbul ignore next */
 		throw new Error(`Unsupported board reference type ${context.type as string}`);
 	}
+
+	private async getFeaturesForBoardContext(context: BoardExternalReference): Promise<BoardFeature[]> {
+		const features: BoardFeature[] = [];
+
+		if (context.type === BoardExternalReferenceType.Course) {
+			const course = await this.courseService.findById(context.id);
+
+			if (
+				this.isVideoConferenceEnabledForCourse(course.features) ||
+				(await this.isVideoConferenceEnabledForSchool(course.school.id)) ||
+				this.isVideoConferenceEnabledForConfig()
+			) {
+				features.push(BoardFeature.VIDEOCONFERENCE);
+			}
+
+			return features;
+		}
+
+		if (context.type === BoardExternalReferenceType.Room) {
+			const room = await this.roomService.getSingleRoom(context.id);
+
+			if ((await this.isVideoConferenceEnabledForSchool(room.schoolId)) || this.isVideoConferenceEnabledForConfig()) {
```

Comment:
also here we get the feature in both cases, even if its only activated for the instance, but not for the school, that does not sound right

Reply:
Good point, we should be able to deactivate the feature for the school even if it's enabled on the instance right?

Reply:
Fixed.